requirements.txt

```diff
@@ -0,0 +1,6 @@
+# For ebill_paynet
+zeep==3.4.0
+# For tests
+freezegun==0.3.12
+vcrpy==2.0.1
+xmlunittest==0.5.0
```

Comment:
Add `jinja2>=2.10` as it was mentioned in a comment above.

Reply:
I did not add it because it is a base requirement for Odoo 12.0 https://github.com/odoo/odoo/blob/44123897329c13fc41801f626616c43a24ca13f5/requirements.txt#L12
But added the comment above because internally we are lagging behind...